Winch.Examples/ExampleItems/Assets/Items/Fish/exampleitems.mccod.json

```diff
@@ -0,0 +1,28 @@
+﻿{
+  "Name": "Raw Cod",
+  "itemNameKey": "exampleitems.mccod.name",
+  "itemDescriptionKey": "exampleitems.mccod.desc",
+  "itemType": "GENERAL",
+  "itemSubtype": "FISH",
+  "sprite": "exampleitems.mccod",
+  "itemColor": {
+    "r": 47,
+    "g": 64,
+    "b": 63,
+    "a": 8
+  },
+  "squishFactor": 1,
+  "value": 5,
+  "dimensions": [
+    "X"
+  ],
+  "minSizeCentimeters": 5,
+  "maxSizeCentimeters": 10,
+  "harvestMinigameType": "DREDGE_RADIAL",
```

Comment:
This shoooould work, but if it fails then try one of the actual fish mini-games. I forgot to change in this pr.

Reply:
I double checked this morning and the game type works so this can be ignored